tests/Unleash.Tests/Variants/StrategyVariantTests.cs

```diff
@@ -0,0 +1,252 @@
+﻿using FakeItEasy;
+using FluentAssertions;
+using NUnit.Framework;
+using System;
+using static Unleash.Tests.Specifications.TestFactory;
+using System.Net;
+using System.Net.Http.Headers;
+using System.Text;
+using Unleash.Internal;
+using Unleash.Scheduling;
+
+using Unleash.Tests.Mock;
+using Unleash.Variants;
+
+namespace Unleash.Tests.Variants
+{
+	public class StrategyVariantTests
+	{
+		[Test]
+		public void Picks_Strategy_Variant()
+		{
+            // Arrange
+            var appname = "test";
+            var toggles = new List<FeatureToggle>
+            {
+                new FeatureToggle(
```

Comment:
I don't know which of those fields matter to the test and which ones happen to satisfy the compiler

Reply:
Thanks - I've tried to make constant values and some methods to improve visibility and overview of parameter usages